res/composer-project.nix

```
@@ -11,16 +11,19 @@
 with lib;
 
 let
-
-  composerPath = <?php echo $composerPath; ?>;
-  cacheEntries = <?php echo $cacheEntries; ?>;
-  localPackages = <?php echo $localPackages; ?>;
+  json = ''
+    {{json}}
+  '';
+  data = builtins.fromJSON (json);
```

The intermediate var doesn't add much, I think?

```suggestion
  data = builtins.fromJSON ''
    {{json}}
  '';
```